ACPs/84-table-preamble/README.md

```diff
@@ -0,0 +1,62 @@
+```text
+ACP: 84
+Title: Table Preamble for ACPs
+Author(s): Gauthier Leonard <https://github.com/Nuttymoon>
+Discussions-To: https://github.com/avalanche-foundation/ACPs/discussions/85
+Status: Proposed
+Track: Meta
+```
+
+## Abstract
+
+The current ACP template features a plain-text code block containing "RFC 822 style headers" as `Preamble` (see [What belongs in a successful ACP?](https://github.com/avalanche-foundation/ACPs?tab=readme-ov-file#what-belongs-in-a-successful-acp)). This header includes multiple links to discussions, authors, and other ACPs.
+
+This ACP proposes to replace the `Preamble` code block with a Markdown table format (similar to what is used in [Ethereum EIPs](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md)).
+
+## Motivation
+
+The current ACPs `Preamble` is (i) not very readable and (ii) not user-friendly as links are not clickable. The proposed table format aims to fix these issues.
+
+## Specification
+
+The following Markdown table format is proposed:
+
+| ACP                            | PR Number                                                                                    |
+| ------------------------------ | -------------------------------------------------------------------------------------------- |
+| **Title**                      | ACP title                                                                                    |
+| **Author(s)**                  | A list of the author's name(s) and optionally contact info: FirstName LastName <foo@bar.com> |
+| **Discussions-To**             | [Hyperlinked GitHub Discussion name](./README.md)                                            |
+| **Status**                     | Proposed, Implementable, Activated, Stale                                                    |
+| **Track**                      | Standards, Best Practices, Meta, Subnet                                                      |
+| **Replaces (\*optional)**      | [Hyperlinked ACP number](./README.md)                                                        |
```

Comment:
To make this new format clearer, I recommend doing `ACP-XX` instead of `Hyperlinked ACP number`